linting/custom_lint.py

```diff
@@ -0,0 +1,79 @@
+import re
+from sphinxlint import *
+from pprint import pformat
+
+# https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections
+HEADINGS_CHARACTERS = [
+    '#', '*', '=', '-', '^', '"', '\'', '+', '`', '\"', '@', '!', ',', '.', '/'
+    # Complete list https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+]
+HEADINGS_REGEX = "(%s)\n" % (
+    "|".join(
+        f"(\\{character}+)" for character in HEADINGS_CHARACTERS
+    )
+)
+HEADINGS_ORDERING_GUIDELINES = [
+    '=', '-', '~', '*', '^',
+]
+
+@checker(".rst")
+def check_headers(file, lines, options=None):
```

Comment:
If I understood correctly, this method is called directly by runbot. We should instead have a `test_files` method that calls `check_headers` among others.

Reply:
Nope, I'm extending sphinx-lint with an additional checker, but I was expecting the runbot to call `python3 custom_lint.py --enable headers` and delegate 99% of the logic to the base sphinx-lint code :eyes: